### PR TITLE
Update reference-design.md

### PR DESCRIPTION
**Pull request by bys1123**
reference:
http://www.st.com/zh/microcontrollers/stm32f427-437.html?querycriteria=productId=LN1789
http://www.st.com/en/microcontrollers/stm32f7x5.html?querycriteria=productId=LN1958

### CODE REVIEW

**Comment by hamishwillee** [comment]
Thanks @bys1123 . @bkueng Can you verify this change?

File: en/debug/reference-design.md [hunk 1]
```diff
@@ -12,8 +12,8 @@ FMU generations 1-3 were designed as open hardware, while FMU generations 4 and
...
-* FMUv4: Pixracer \(STM32F427, 168 MHz, 192 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
-* FMUv4 PRO: Drotek Pixhawk 3 PRO \(STM32F467, 168 MHz, 384 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
-* FMUv5: final name TBD \(STM32F7675, 200 MHz, 512 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165)\)
+* FMUv3: Pixhawk variants with 2MB flash \(3DR Pixhawk 2 \(Solo\), Hex Pixhawk 2.1, Holybro Pixfalcon, 3DR Pixhawk Mini, STM32F427, 180 MHz, 256 KB RAM, 2 MB flash, [schematics](https://github.com/PX4/Hardware/tree/master/FMUv2)\)
+* FMUv4: Pixracer \(STM32F427, 180 MHz, 256 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\) (Wrong Link)
```
Comment (bkueng) [inline en/debug/reference-design.md]: Why did you add 'Wrong Link' here?

**Review by bkueng — COMMENTED** [review]
The changes look correct, thanks @bys1123.

File: en/debug/reference-design.md (hunk 1 above)
Comment (bys1123) [reply en/debug/reference-design.md]: Looks like it only have 176-pin STM32F469IIT6 Signal pinout. But Pixracer is using STM32F427. Is that FMUv4-Mini the right one? Should we add some explanation at that cell.

Comment (bkueng) [reply en/debug/reference-design.md]: I see, you're right, Pixracer is using STM32F427. @davids5 can you help clarify that?

Comment (davids5) [reply en/debug/reference-design.md]: The RAM change is correct.  The speed is incorrect. The CPU is capable of running at 180Mhz. The PX4 Firmware project uses USB and due to the F4's PLL limitations, that limits the the  CPU to running at 168 MHz.

File: en/debug/reference-design.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ FMU generations 1-3 were designed as open hardware, while FMU generations 4 and
...
+* FMUv4 PRO: Drotek Pixhawk 3 PRO \(STM32F467, 180 MHz, 384 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
```
Comment (davids5) [inline en/debug/reference-design.md]: FMUv4 PRO: Drotek Pixhawk 3 PRO uses the  STM32F469IIT6

File: en/debug/reference-design.md (hunk 1 above)
Comment (davids5) [reply en/debug/reference-design.md]: That is the the V4Pro pinout. But documents V2 and V4. I  was supposed be a comparison, with V2, V4 and V4 Pro I updated it with the SoC's part numbers for clarity

File: en/debug/reference-design.md [hunk 3]
```diff
@@ -12,8 +12,8 @@ FMU generations 1-3 were designed as open hardware, while FMU generations 4 and
...
+* FMUv5: Holybro Pixhawk 4 \(STM32F765, 216 MHz, 512 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1-n0__BYDedQrc_2NHqBenG1DNepAgnHpSGglke-QQwY/edit#gid=912976165)\)
```
Comment (davids5) [inline en/debug/reference-design.md]: FMUv5  uses STM32F765IIK

**Review by davids5 — CHANGES_REQUESTED** [review]
If the FMUV4xx pinout now make sense please remove the "(Wrong Link)"

File: en/debug/reference-design.md [hunk 4]
```diff
@@ -12,8 +12,8 @@ FMU generations 1-3 were designed as open hardware, while FMU generations 4 and
...
-* FMUv4: Pixracer \(STM32F427, 180 MHz, 256 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\) (Wrong Link)
-* FMUv4 PRO: Drotek Pixhawk 3 PRO \(STM32F467, 180 MHz, 384 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
+* FMUv3: Pixhawk variants with 2MB flash \(3DR Pixhawk 2 \(Solo\), Hex Pixhawk 2.1, Holybro Pixfalcon, 3DR Pixhawk Mini, STM32F427, 168 MHz, 256 KB RAM, 2 MB flash, [schematics](https://github.com/PX4/Hardware/tree/master/FMUv2)\)
+* FMUv4: Pixracer \(STM32F427, 168 MHz, 256 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\) 
+* FMUv4 PRO: Drotek Pixhawk 3 PRO \(STM32F467, 168 MHz, 384 KB RAM, 2 MB flash, [pinout](https://docs.google.com/spreadsheets/d/1raRRouNsveQz8cj-EneWG6iW0dqGfRAifI91I2Sr5E0/edit#gid=1585075739)\)
```
Comment (davids5) [inline en/debug/reference-design.md]: STM32F467 should be STM32F469,  The clock rate on the F46 family is 180 Mhz (this Soc uses a different PLL then the F42xx)

**Review by davids5 — CHANGES_REQUESTED** [review]
see inline comment

**Review by davids5 — APPROVED** [review]
@bys1123 - Thank you!

**Comment by hamishwillee** [comment]
Thanks all. Merging.

File: en/debug/reference-design.md (hunk 4 above)
Comment (bys1123) [reply en/debug/reference-design.md]: Thanks for your explanation. Very helpful.
